DailyKanban/Views/CreateItemView.swift

```diff
@@ -20,20 +20,18 @@ struct CreateItemNavigationView: View {
     
     /// Stepper
     @State var numTodos: Int = 0
-    private let maxTodos: Int
-    @State var todos: [String]
+    private var maxTodos: Int = 0
+    @State var todos: [String] = []
```

Comment:
I think a good rule of thumb is to have a default value for state and published variables?

Reply:
Yeah, sure, idk why that blocked you from compiling tho

Reply:
Like the array sure... But the constant??